v1/rest/responsestruct.go

```diff
@@ -65,3 +65,9 @@ type UnprocessableEntityResponse struct {
 	Error string `json:"error" example:"Unprocessable Entity"`
 	message
 }
+
+// 423 Locked
+type LockedResponse struct {
+	Error string `json:"error" example:"Status Locked"`
```

Comment:
432 message `Locked` saja
```suggestion
	Error string `json:"error" example:"Locked"`
```